ctapipe_io_magic/__init__.py

```diff
@@ -80,6 +82,78 @@
     DATA_MAGIC_LST_TRIGGER: EventType.SUBARRAY,
 }
 
+class ReportLaserContainer(Container):
+    """ Container for Magic laser parameters """
+    UniqueID = Field(None, 'No.')
+    Bits = Field(None, 'ID')
+    MJD = Field(None, 'Modified Julian Date')
```

Comment:
the time information is stored different in Drive and Laser reports. Both of them have MTime object where the MJD is divided into the integer number and millisec and nanosec, but Drive report has also a separete field with full MJD. In Laser report I do not see such a field, so the MJD information should be constructed from the integer MJD and milisec (nanosec I think you can skip, the laser shot is long enough that milisec precision is perfectly fine).
So in the container you should only have MJD with the time format and drop milisec and nanosec  (and see my comments below)